lib/ohai/config.rb

```diff
@@ -40,6 +40,7 @@ class Config
       default :run_all_plugins, false
       # optional plugins are the set of plugins that are marked optional but you wish to run.
       default :optional_plugins, []
+      default :timeout, 30
```

Comment:
Can you change this to shellout_timeout so it's more clear what it changes?

Reply:
+1 on using a more specific name

Reply:
Fair point, will do.